src/promptflow-devkit/promptflow/_sdk/operations/_trace_operations.py

```diff
@@ -185,14 +196,25 @@ def list_line_runs(
             session_id=session_id,
             line_run_ids=line_run_ids,
         )
-        line_runs = []
-        for obj in orm_line_runs:
-            line_run = LineRun._from_orm_object(obj)
-            orm_eval_line_runs = ORMLineRun._get_children(line_run_id=line_run.line_run_id)
-            eval_line_runs = [LineRun._from_orm_object(obj) for obj in orm_eval_line_runs]
-            line_run._append_evaluations(eval_line_runs)
-            line_runs.append(line_run)
-        return line_runs
+        return self._parse_line_runs_from_orm(orm_line_runs)
+
+    def _search_line_runs(
+        self,
+        expression: str,
+        collection: typing.Optional[str] = None,
+        runs: typing.Optional[typing.Union[str, typing.List[str]]] = None,
```

Comment:
what's the expected format when runs is a str?

Reply:
str represents single run, and i will make it a list of str